src/scss/_general.scss

```diff
@@ -0,0 +1,67 @@
+body {
+  margin: 0;
```

Comment:
Du definerer marginer og padding i sass. Det kan hende det kan brukes en gang eller to i løpet av et stort prosjekt. Men poenget er å bruke classes som p-0 og m-0 på body i stedet.